src/components/CompanyHome.jsx

```diff
@@ -14,48 +12,42 @@ class CompanyHome extends Component {
 
     componentDidMount() {
         let company = localStorage.getItem('company')
-        db.collection('companies').doc(company).collection('forms')
+        db.collection('companies').doc(company).collection('Forms')
             .get()
             .then(snapshot => {
                 let companyData = []
                 snapshot.forEach(doc => {
                     companyData.push(doc.data())
                 });
-                this.setState({companyData})
+                this.setState({ companyData })
             })
     }
 
     render() {
+        let result, company;
+        if (this.state.companyData) {
+            result = Object.keys(this.state.companyData[0]).map(data => ({ [data]: this.state.companyData[0][data] }))
```

Comment:
In case you were curious the for ... in version would look like this

`for(key in this.state.companyData) {
     result.push({key: this.state.companyData[key]})
  }`

Reply:
Are we planning to use 'results' in this page?